scripts/uniffi_bindgen_generate_kotlin_android.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+BINDINGS_DIR="bindings/kotlin"
+TARGET_DIR="target"
+PROJECT_DIR="ldk-node-android"
+PACKAGE_DIR="org/lightningdevkit/ldknode"
+UNIFFI_BINDGEN_BIN="cargo +nightly run --features=uniffi/cli --bin uniffi-bindgen"
+ANDROID_NDK_ROOT="/opt/homebrew/share/android-ndk"
+LLVM_ARCH_PATH="darwin-x86_64"
+PATH="$ANDROID_NDK_ROOT/toolchains/llvm/prebuilt/$LLVM_ARCH_PATH/bin:$PATH"
+
+rustup +nightly target add x86_64-linux-android aarch64-linux-android armv7-linux-androideabi
+#cargo build --release || exit 1
+CFLAGS="-D__ANDROID_API__=21" AR=llvm-ar CARGO_TARGET_X86_64_LINUX_ANDROID_LINKER="x86_64-linux-android21-clang" CC="x86_64-linux-android21-clang" cargo +nightly build --profile release-smaller --target x86_64-linux-android || exit 1
+CFLAGS="-D__ANDROID_API__=21" AR=llvm-ar CARGO_TARGET_ARMV7_LINUX_ANDROIDEABI_LINKER="armv7a-linux-androideabi21-clang" CC="armv7a-linux-androideabi21-clang" cargo +nightly build --profile release-smaller --target armv7-linux-androideabi || exit 1
+CFLAGS="-D__ANDROID_API__=21" AR=llvm-ar CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER="aarch64-linux-android21-clang" CC="aarch64-linux-android21-clang" cargo +nightly build --profile release-smaller --target aarch64-linux-android || exit 1
```

Comment:
looks like these lines are replicating what `bdk-android` is [doing here](https://github.com/bitcoindevkit/bdk-ffi/blob/c1243f9e1c9ff6b5661ac54d799fc2f0d96232a0/bdk-android/plugins/src/main/kotlin/org/bitcoindevkit/plugins/UniFfiAndroidPlugin.kt)?